sdk-python/omnilogger/logger.py

```diff
@@ -18,48 +18,60 @@ def start_listener(url: str):
     sys.stderr = StreamToLogger(logger, logging.ERROR)
 
 
-def handle_openai_response(data: str, url: str):
-    output = data.split("body='")[1].split("' headers=")[0]
-    output = bytes(output, "utf-8").decode("unicode_escape")
-    output = json.loads(output)
-
-    log = {
-        "input": "This is a new dummy input!",
-        "datetime_utc": datetime.utcfromtimestamp(output["created"]),
-        "output": output["choices"][0]["text"],
-        "total_tokens": output["usage"]["total_tokens"],
-    }
-    send_to_db(url, log)
-
-
 class OpenAIFilter(logging.Filter):
     url: str
+    t_prompt: str
```

Comment:
what does t_ mean?

Reply:
I use t_ for temporary variables, but it might be confusing so I'll revert it